itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/history/TestHiveHistory.java

```diff
@@ -146,23 +146,29 @@ public void testSimpleQuery() {
       if (ret != 0) {
         fail("Failed");
       }
-      HiveHistoryViewer hv = new HiveHistoryViewer(SessionState.get()
-          .getHiveHistory().getHistFileName());
-      Map<String, QueryInfo> jobInfoMap = hv.getJobInfoMap();
-      Map<String, TaskInfo> taskInfoMap = hv.getTaskInfoMap();
-      if (jobInfoMap.size() != 1) {
-        fail("jobInfo Map size not 1");
-      }
 
-      if (taskInfoMap.size() != 1) {
-        fail("jobInfo Map size not 1");
-      }
+      // Disabled HiveHistoryViewer feature if history-scribe is enabled.
```

Comment:
I'd rather not doing it in this way.

Just change the test session property `hive.session.history.scribe.enabled` to false and use this code to test logging to local file system.